go.mod

```diff
@@ -322,6 +322,7 @@ require (
 )
 
 replace (
+	github.com/celestiaorg/nmt => github.com/celestiaorg/nmt v0.12.0 // temporary patch until we either fix nmt v0.13.0 or downgrade across all repos
```

Comment:
as mentioned in the comment, we can either downgrade across all repos, or simply remove this after we fix whatever is breaking v0.13.0

Reply:
We decided to remove replace as we fixed the issue with nmt and updated it in https://github.com/celestiaorg/celestia-node/pull/1651